packages/angular/src/froala/plugins/citationManager.ts

```diff
@@ -0,0 +1,132 @@
+import {cookie} from '@stratusjs/core/environment'
+import {Stratus} from '@stratusjs/runtime/stratus'
+// Universal Button
+import {
+    InputButtonPlugin
+} from '@stratusjs/angular/froala/plugins/inputButton'
+
+// @ts-ignore
+import FroalaEditor from 'froala-editor'
+
+
+// Local Setup
+const systemDir = '@stratusjs/angular'
+const moduleName = 'citationManager'
+const parentModuleName = 'froala/plugins'
+
+// Directory Template
+const min = !cookie('env') ? '.min' : ''
+const localDir = `${Stratus.BaseUrl}${boot.configuration.paths[`${systemDir}/*`].replace(/[^/]*$/, '')}`
+
+// Plugin Options
+FroalaEditor.DEFAULTS = Object.assign(FroalaEditor.DEFAULTS, {
+    endpoint: '/Api/Content'
+})
+
+/**
+ * @param editor The Froala instance
+ */
+FroalaEditor.PLUGINS.citationManager = function citationManager (editor: any) {
+    let inputButton: InputButtonPlugin
+
+    const debug = true // needs false
+
+    // When the plugin is initialized,this will be called.
+    function _init() {
+        Stratus.Internals.CssLoader(`${localDir}${parentModuleName}/${moduleName}${min}.css`)
+        inputButton = new InputButtonPlugin({
+            name: 'Citation Manager',
+            eventName: 'citation-input',
+            editor,
+            autoSaveSelection: true,
+            autoRestoreSelection: true,
+            debug
+        })
+
+        if (debug) {
+            console.log('initialized:', {
+                options: editor.opts.endpoint,
+                instance: this
+            })
+        }
+
+        // Editor methods
+        // editor.methodName(params);
+
+        // Event listeners
+        // editor.events.add('contentChanged', function (params) {});
+    }
+
+    function onClick() {
+        if (!editor.el) {
+            console.warn('citationManager.onClick(): unable to find element')
+            return
+        }
+        inputButton.onClick(editor.el)
+    }
+
+    // Expose public methods.
+    // Public methods can be accessed through the editor API:
+    // editor.myPlugin.publicMethod();
+    return {
+        // If _init is not public then the plugin won't be initialized.
+        _init,
+        onClick
+    }
+}
+
+// Insert Plugin to Image Insert TODO will not in the end
+FroalaEditor.DEFAULTS.imageInsertButtons.push('citationManager')
```

Comment:
This line is unnecessary for this plugin, but carried over from `mediaManager.ts`.

Reply:
(actually an accidental leftover when i had it inserted in every possible location because it wasn't loading)